### ISSUE TITLE
A11y shrine: replace background logo

### ISSUE THREAD

**Issue by felixzapata** [PolymerElements/app-layout#358]
### Description

The logo is a background image but there is no alternative text neither a replacement text technique.
### Expected outcome

The logo should be an image element or at least to have the name of the project.
### Actual outcome

The logo is a css background image.
